STANDARDS.md

```diff
@@ -70,7 +74,7 @@ A revision means creating a new dataset based on an existing one, with the inten
 
 ## Dataset naming and versioning
 
-Each dataset shall be versioned.
+Each dataset shall be versioned according to `vX.Y.Z`. X refers to the major version, Y to the minor version, and Z to a micro version.
 - The naming of a dataset should have the following structure:
 
     `"OpenFF <descriptive and uniquely-identifying name> v<version number>"`
```

Comment:
See question above about whether we want to update versioning to start at 4, or keep it at 3